src/SonarScanner.MSBuild.Tasks/GetAnalyzerSettings.cs

```diff
@@ -326,31 +327,59 @@ private AnalyzerSettings GetLanguageSpecificSettings(AnalysisConfig config)
 
         /// <summary>
         /// Merges and returns the supplied list of file paths. In case of duplicate
```

Comment:
this is not really looking for duplicate file names, but _analyzer names that contain "SonarAnalyzer"_.

Test case:

> Analyzer Include='should.be.removed.SonarAnalyzer.Fake.dll' 

This name is not duplicate of one of our analyzers, it just happens to contain the `sonaranalyzer` string inside, so the comment should be updated.